### DISCUSSION TITLE
建了一个微信群欢迎大家进群交流

### DISCUSSION THREAD

**Discussion by shitoudada** [taowen/awesome-lowcode#80] [General]
欢迎对低代码感兴趣的朋友们入群
![qrcode_for_gh_6d1dfde95270_258](https://user-images.githubusercontent.com/10807769/157180826-b0f994c0-8a6b-4946-a521-763d81c93c4a.jpg)

**Comment by qmdx** [comment]
[开源低代码，前来报道 ](http://aizuda.com/)

**Comment by LowCoderGitHub** [comment]
搞了个低代码行业的交流平台，大佬们有没有兴趣来新平台玩玩
https://www.lowcoder.com.cn/
